src/openrct2/world/Sprite.h

```diff
@@ -138,7 +138,7 @@ union rct_sprite
     rct_money_effect* AsMoneyEffect();
     Peep* AsPeep();
 };
-assert_struct_size(rct_sprite, 0x100);
+assert_struct_size(rct_sprite, 0x200);
```

Comment:
Out of curiosity, I thought that `structs` with these asserts couldn't be changed due to export/import limitations, isn't that true?

Reply:
Not quite true. These are the in memory structs. We can change their sizes and structure but any new fields can't be saved as that would break export/import.

Reply:
Ah, alright, thanks for explaining :)